package.json

```diff
@@ -50,6 +50,7 @@
     "@openmrs/esm-error-handling": "3.x",
     "@openmrs/esm-react-utils": "3.x",
     "carbon-components": "^10.21.0",
+    "@carbon/icons-react": "^10.25.0",
```

Comment:
These are not shared afaik. We only share the framework-independent carbon packages.